src/extension/content-script/onendwebln.js

```diff
@@ -33,7 +33,11 @@ async function init() {
     return;
   }
 
-  injectScript(browser.runtime.getURL("js/inpageScript.bundle.js")); // registers the DOM event listeners and checks webln again (which is also loaded onstart
+  const providerScript = "@@@WINDOW_PROVIDER@@@";
```

Comment:
this is only used if (!isMv3) right? can you move it into the if block?

Reply:
resolved